backstop/tests/carousel.html

```diff
@@ -0,0 +1,355 @@
+                  href="#slide-2"
+                  id="test-1"
+                ></a>
+              </li>
+              <li>
+                <a
+                  class="iui-carousel-navigation-dot"
+                  href="#slide-3"
+                  id="test-2"
+                ></a>
+              </li>
+            </ol>
+          </div>
+          <button
+            class="iui-button iui-borderless iui-small iui-carousel-navigation-button"
+            aria-controls="myCarousel-items"
+            aria-label="Next slide"
+          >
+            <svg-chevron-right
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-chevron-right>
+          </button>
+        </nav>
+      </div>
+    </section>
+
+    <h3>Many slides</h3>
+    <p class="iui-text-small iui-text-muted">The dots are not functional in this demo.</p>
+    <section id="demo-many-slides">
+      <div
+        class="iui-carousel"
+        aria-roledescription="carousel"
+        aria-label="Introduction to iTwinUI"
+        aria-live="polite"
+      >
+        <div
+          class="iui-carousel-slider"
+          id="myCarousel-items-3"
+        >
+
+          <div
+            class="iui-carousel-slider-item"
+            style="background-image: url('../assets/image-test.jpg');"
+          >
+            <div class="iui-carousel-slider-item-title">Bentley System headquarters
+            </div>
+          </div>
+
+          <div
+            class="iui-carousel-slider-item"
+            style="background-color: var(--iui-color-background-4);"
+          >2</div>
+
+          <div
+            class="iui-carousel-slider-item"
+            style="background-color: var(--iui-color-background-5);"
+          >3</div>
+
+        </div>
+
+        <nav class="iui-carousel-navigation">
+          <button
+            class="iui-button iui-borderless iui-small iui-carousel-navigation-button"
+            aria-controls="myCarousel-items-3"
+            aria-label="Previous slide"
+          >
+            <svg-chevron-left
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-chevron-left>
+          </button>
+          <div class="iui-carousel-navigation-container">
+            <ol class="iui-carousel-navigation-dots">
+              <li>
+                <span class="iui-carousel-navigation-dot iui-invisible"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot iui-first"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot iui-second"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot iui-active"></span>
```

Comment:
these are not focusable, have no cursor pointer.. expected?
also clicking gets messy

![image](https://user-images.githubusercontent.com/81580355/147753278-2b1508c9-4d55-4db1-9d73-86c39373e5e8.png)

Reply:
The first carousel is using `<a>` so they have the hand cursor and are in the tabindex.  The second carousel is demonstrating that you can replace the `<a>` with a `<span>` if the user doesn't want the dots to be clickable.  The JS to change what dot is active is incorrectly applying to the `<span>`s as well, so that could be fixed.  I do have a warning on the demo page that says "The dots are not functional in this demo."

Reply:
But does it make sense to have dot not clickable?

Reply:
also.. off topic, but I just thought, that now your sliding in carousel is working through scroll, if I understood correctly..
I am just wondering, that to sync this together with the dots below using js (if you use touch to slide), might be more complicated...

Reply:
I am unsure if we want to have non-actionable dots or not.  We can force them to always be actionable if you'd like.

Reply:
And to your second thought, I am very iffy about the overflow scroll implementation.  I discovered that if you click on the dots, the whole page scrolls down to the top of the carousel which probably isnt ideal.
![carousel](https://user-images.githubusercontent.com/849817/147770310-eb6cf1ed-5f0c-490e-89f2-7b51074f1a1a.gif)